docs/app/documentation/component-docs/shellbar/examples/shellbar-collapsible-example.component.html

```diff
@@ -10,19 +10,16 @@
     <fd-shellbar-subtitle>
         Subtitle
     </fd-shellbar-subtitle>
-    <fd-copilot></fd-copilot>
     <fd-shellbar-actions [user]="user"
                          [userMenu]="userMenu"
                          [productSwitcher]="productSwitcher">
-        <button fd-button
-                [fdType]="'standard'">Custom Button</button>
```

Comment:
It is a good to have an example with a custom component inserted in the shellbar